macvtap.go

```diff
@@ -197,16 +213,24 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	if n.MAC != "" {
+	envArgs, err := getEnvArgs(args.Args)
+	var mac net.HardwareAddr
+	if envArgs != nil {
```

Comment:
What if you get some env args other then MAC. Then you go forward, parse MAC, fail and exit the application.

Maybe I'm missing something, but can it mean that with this approach, you require users to always specify MAC?

Reply:
I don't think so; there's a test to check that in https://github.com/maiqueb/macvtap-cni/pull/2/files#diff-1b8ca11f62a2e5f6906d9deb90536c42R178 .

My "go" is pretty limited, so I might be wrong.

Reply:
Well, let's trust the test. If it is indeed an issue, it would appear pretty soon.